appveyor.yml

```diff
@@ -107,7 +107,7 @@ for:
 
 # Create Installers to store at appveyor and upload to github
 after_test:
-  - cmd: iscc /O "Rubberduck.Deployment\InnoSetup\Rubberduck.Installer.Build.iss"
+  - cmd: "C:\Program Files (x86)\Inno Setup 6\iscc.exe" /O "Rubberduck.Deployment\InnoSetup\Rubberduck.Installer.Build.iss"
```

Comment:
cinst innosetup installs an exe to the program files directory? the Windows ecosystem is a mess :stuck_out_tongue_winking_eye: